test/e2e/adminUI/tests/group005Fields/commonFieldTestUtils.js

```diff
@@ -0,0 +1,128 @@
+module.exports = {
+	beforeUI: function (browser) {
+		browser
+		browser.app = browser.page.app();
+		browser.signinPage = browser.page.signin();
+		browser.listPage = browser.page.list();
+		browser.initialFormPage = browser.page.initialForm();
+
+		browser.app.navigate();
+		browser.app.waitForElementVisible('@signinScreen');
+
+		browser.signinPage.signin();
+		browser.app.waitForElementVisible('@homeScreen');
+	},
+	beforeUX: function (browser) {
```

Comment:
I would consolidate beforeUI and beforeUX..I don't think it hurts anything to have the item page included in both cases.